### ISSUE TITLE
[PAN-OS] Subversions removed by automation

### ISSUE THREAD

**Issue by lryanuk** [endoflife-date/release-data#26]
Appears to be happening since #20 

https://github.com/endoflife-date/endoflife.date/commit/7b8d741ba7c12a902686ec8f48486d48844eb4b7#diff-4e56730cfbfc9ec6c96e6e7a7501187ccc05b55964efac84b70237008cb7b2c4

https://github.com/endoflife-date/endoflife.date/commit/8c5a5135d8805980f75862beaf3b60f41a5ee16d#diff-4e56730cfbfc9ec6c96e6e7a7501187ccc05b55964efac84b70237008cb7b2c4

**Comment by captn3m0** [comment]
Yes, that's a side-effect of relying on the [PAN EOL](https://www.paloaltonetworks.com/services/support/end-of-life-announcements/end-of-life-summary) page for automation, which doesn't have patch versions noted.

Getting those from the docs website is very scattered: https://docs.paloaltonetworks.com/pan-os/10-2/pan-os-release-notes/pan-os-10-2-3-known-and-addressed-issues, any suggestions for a better source?

**Comment by captn3m0** [comment]
Did a bit of research on this, this could be scraped, but there is no clear date of release. The closest is the date the Known Issues page for a patch release was updated, and that isn't exactly reliable.

**Comment by lryanuk** [comment]
I haven't been able to find a reliable source that's publicly available for release date, however as a customer I have access to this data via the product. I'm happy to maintain latest release data and scrape the links periodically until an automated solution is found?

Latest release data from the product below for PAN-OS, also have access to GlobalProtect app:
![image](https://user-images.githubusercontent.com/13711306/210439151-0666aae2-1afa-4a0b-b403-ccd202998196.png)

**Comment by captn3m0** [comment]
@lryanuk Thanks! PRs filed directly to the endoflife.date page (with `latest/latestReleaseDate` field) should work (The automation doesn't downgrade versions).

Maintaining the `release-data` itself manually will break things, since we have automation that will undo your changes. It's also probably better to keep this repo completely automated.
